### ISSUE TITLE
Add Maven support

### ISSUE THREAD

**Issue by opperpanter** [pieroxy/java-user-agent-detection#7]
Hi,

All our projects are built with Maven, so for use it would be very convenient to have the library available via Maven (Central).

Valentijn

**Comment by pieroxy** [comment]
I've been meaning to do it. I wanted to wait until the lib is stable enough, but I guess now is as good a time as any.

**Comment by pieroxy** [comment]
I'll probably do it this way: http://central.sonatype.org/pages/ossrh-guide.html

But first I have a moderately big breaking change to pass. Hang on for a few weeks.

**Comment by mrDoctorWho** [comment]
Hi,

Any progress on this? It's been too long _few weeks_, I can tell.

**Comment by pieroxy** [comment]
That would be perfect.

**Comment by pieroxy** [comment]
@opperpanter Things are moving, the lib is not maven compatible.

**Comment by ceefour** [comment]
Hope to see this on Maven Central :) Excellent work 👍

**Comment by hth** [comment]
I agree. Awesome library. Great Work!!!! some day it will be on Maven. But not this month.

**Comment by mrDoctorWho** [comment]
I suppose, the issue can be now closed?

**Comment by xjodoin** [comment]
The lib is not on maven central
